generate.py

```
@@ -0,0 +1,394 @@
+            if validation_result['valid']:
+                click.echo("✅ Response validated successfully!")
+                
+                # Save validated draft
+                save_validated_draft(city, vibe, validation_result['parsed_data'], prompt_version)
+                
+                # Update monitor
+                monitor.mark_as_generated(city, vibe)
+                
+                # Show summary
+                click.echo(f"  ✓ Restaurant count: {validation_result['restaurant_count']}")
+                click.echo(f"  ✓ All required fields present")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)